vpdq/python/vpdq.pyx

```diff
@@ -117,8 +118,16 @@ def computeHash(
     Returns:
         list of vpdq_feature: VPDQ hash from the video
     """
+    if not Path(input_video_filename).is_file():
```

Comment:
If you do want to take either union or str, strongly suggest normalizing to one in the first line. I.e.

```
input_video_filename = Path(input_video_filename )
```

This will make tracking what type you expect much easier

Reply:
Thanks for the suggestion! Have added this.